### PR TITLE
#159987636 Implement article share on social media

### PR DESCRIPTION
**Pull request by emekafredy**
#### What does this PR do?
- Implement article share on social media

#### Description of Task to be completed?
- Set up a package to allow users share articles on facebook, twitter, and through their email

#### How should this be manually tested?
- clone the repo
- cd into 'thor-ah-frontend’
- checkout to the branch ```ft-share-article-on-social-media-159987636```
- add the app URL ```https://author-haven-staging.herokuapp.com ``` to your .env with the key ```REACT_APP_WEB_URL```
- run ```npm install```
- run ```npm run dev```
- click on an article to try the share buttons.

#### What are the relevant pivotal tracker stories?
[159987636](https://www.pivotaltracker.com/story/show/159987636)

#### Screenshots (if appropriate)
screen shot 2018-10-24 at 8 31 46 am

#### Questions:

### CODE REVIEW

**Review by gonuoha — APPROVED** [review]
LGTM

File: public/index.html [hunk 1]
```diff
@@ -20,4 +20,5 @@
   <div id="root">
   </div>
   <div id='modal'></div>
+  <!-- <script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-5bd0171290e858e6"></script> -->
```
Comment (d-beloved) [inline public/index.html]: You may want to remove this code.

**Review by d-beloved — APPROVED** [review]
LGTM

File: public/index.html (hunk 1 above)
Comment (emekafredy) [reply public/index.html]: Okay. Thanks for noticing this

**Review by johnkingzy — APPROVED** [review]
LGTM

**Pull request merged**